benchmarks/cluster3nodes/run_tx_generator.sh

```diff
@@ -28,7 +28,6 @@ run 'cardano-tx-generator' \
   --outputs-per-tx $outputstx \
   --tx-fee $txfee \
   --tps $tps \
-  --single-threaded \
```

Comment:
ah, now you remove "single-threaded" again

Reply:
Right, for the period of time when we had the single-threaded submission, I thought it wise to recreate the cluster's conditions in the local testing environement.